packages/abstract-signer/src.ts/index.ts

```diff
@@ -199,6 +194,22 @@ export abstract class Signer {
         }
 
         const tx = shallowCopy(transaction);
+        if (!tx.nodeId) {
+            let nodeID : any;
+            if (transaction.nodeId) {
+                nodeID = transaction.nodeId;
+            } else {
+                this._checkProvider();
+                // provider present, we can go on
+                const submittableNodeIDs = this.provider.getHederaNetworkConfig();
+                if (submittableNodeIDs.length > 0) {
+                    nodeID = submittableNodeIDs[0];
```

Comment:
Instead of hardcoding the nodeId we will target, we should use the `hedera client` method for getting that node Id. I know that they are choosing one node at random that they target. That would make the submission more resilient